src/components/EmployeeModalForm/EmployeeCreateForm.tsx

```diff
@@ -49,41 +50,54 @@ export const EmployeeCreateForm: Component<Props> = ({ state, actions }) => {
             <FormInput
               name={firstName.name}
               value={firstName.value}
+              maxLength={INPUT_LENGTH_SHORT_MAX}
               type="text"
+              hasError={firstName.errors != null && firstName.errors.required}
               onInputChange={formActions.updateValue}
             />
+            {firstName.errors != null && firstName.errors.required && <FormHint theme="danger" label="First name is required" />}
           </FormField>
           <FormField labelText="Last Name" required={true}>
             <FormInput
               name={lastName.name}
               value={lastName.value}
+              maxLength={INPUT_LENGTH_SHORT_MAX}
               type="text"
+              hasError={lastName.errors != null && lastName.errors.required}
               onInputChange={formActions.updateValue}
             />
+            {lastName.errors != null && lastName.errors.required && <FormHint theme="danger" label="Last name is required" />}
           </FormField>
           <FormField labelText="Email" required={true}>
             <FormInput
               name={emailAddress.name}
               value={emailAddress.value}
+              maxLength={INPUT_LENGTH_LONG_MAX}
               type="email"
+              hasError={emailAddress.errors != null && (emailAddress.errors.required || emailAddress.errors.email)}
               onInputChange={formActions.updateValue}
             />
+            {emailAddress.errors != null && emailAddress.errors.required && <FormHint theme="danger" label="Email is required" />}
+            {emailAddress.errors != null && emailAddress.errors.email && <FormHint theme="danger" label="Email invalid" />}
           </FormField>
           <FormField labelText="Password" required={true}>
             <FormInput
               name={password.name}
               value={password.value}
               type="password"
+              hasError={password.errors != null && password.errors.required}
               onInputChange={formActions.updateValue}
             />
+            {password.errors != null && password.errors.required && <FormHint theme="danger" label="Password is required" />}
           </FormField>
-          <FormField labelText="Status" required={true} hint="Inactive: No login possible">
+          <FormField labelText="Status" required={true}>
```

Comment:
You might want to add the errors to the FormCheckbox too.

Reply:
Checkbox has no styling options